app/controllers/namespaces_controller.rb

```diff
@@ -46,6 +46,24 @@ def create
     end
   end
 
+  # PATCH/PUT /namespace/1
+  # PATCH/PUT /namespace/1.json
+  def update
+    authorize @namespace
+    namespace_params = params.require(:namespace).permit(:description)
+    old_description = @namespace.description
+    @namespace.update(namespace_params)
+
+    @namespace.create_activity :change_namespace_description,
+                               owner:             current_user,
+                               recipient:         @namespace,
+                               parameters:        {
+                                 old_description: old_description,
+                                 new_description: namespace_params["description"] }
+
+    render js: "window.location = '/namespaces/#{@namespace.id}'"
```

Comment:
Shall I limit the length of the description? The view of the activity shows not the value of the description. Only if it has been edited or deleted. Only the csv contains this information. So, I guess I can remove the parameters?